_includes/head.html

```diff
@@ -27,6 +27,10 @@
 <meta property="og:description" content="{% if page.description %}{{ page.description | escape }}{% else %}{{ site.description }}{% endif %}">
 <meta property="og:image" content="{% if page.image %}{{ page.image | absolute_url }}{% else %}{{ site.default_img | absolute_url }}{% endif %}">
 
+<!-- Lightning Address for Alby's Value4Value extensions -->
+{% capture shortName %}{{ page.redirect_from | replace: '/', '' }}{% endcapture %}
+{% capture lightningAddress %}{% if shortName %}{{ shortName }}{% else %}{{ 's@ts.dergigi.com' }}{% endif %}{% endcapture %}
+<meta name="lightning" content="lnurlp:{{ lightningAddress }}"/>
```

Comment:
I wanted to send some sats and test it, but it seems your `lightningAddress` is empty here. jfyi. :)

Reply:
ich habe keine Ahnung von liquid templates, aber muss das sowas sein?

```
{% capture lightningAddress %}{% if shortName != '' %}{{ shortName | append: "@ts.dergigi.com" }}{% else %}{{ 's@ts.dergigi.com' }}{% endif %}{% endcapture %}
```

also wenn der `shortName` nicht ein empty string ist (der shortName wird auf `''` gesetzt und ist deshalb bisher truthy) dann soll die addresse `{shortName}@ts.dergigi.com` sein, ansonsten `s@ts.dergigi.com`

Reply:
vielleicht auch:

```
{% capture lightningAddress %}{% if shortName == blank %}{{ 's@ts.dergigi.com' }}{% else %}{{ shortName | append: "@ts.dergigi.com" }}{% endif %}{% endcapture %}
```

Reply:
Thank you! Should be fixed with #288 - might take a minute or two until it is live. Tested it on local, and it truly should be fixed now.